pkg/k8s/api/outdated.go

```diff
@@ -0,0 +1,93 @@
+package api
+
+import (
+	"encoding/json"
+	bolt "go.etcd.io/bbolt"
+	"golang.org/x/xerrors"
+	"io"
+	"path/filepath"
+
+	"github.com/aquasecurity/trivy-db/pkg/db"
+	"github.com/aquasecurity/trivy-db/pkg/types"
+	"github.com/aquasecurity/trivy-db/pkg/utils"
+)
+
+const (
+	k8sOutdatedAPiDir                = "api"
+	K8sOutdatedAPI    types.SourceID = "outdated-api"
+)
+
+type Advisory types.OutDatedAPIData
```

Comment:
Is this an advisory?

Reply:
its a document dB not security , happy to change it